src/components/Composer.vue

```diff
@@ -302,6 +302,9 @@ export default {
 
 		this.bodyVal = this.bodyWithSignature(this.selectedAlias, this.bodyVal)
 	},
+	mounted() {
+		this.$refs.toLabel.$el.focus()
```

Comment:
Guess ``this.$refs.toLabel.focus()`` wouldn't work because it's a Vue component and not a DOM element. Nice trick :+1: